editor/components/page-attributes/order.js

```diff
@@ -28,24 +28,33 @@ export function PageAttributesOrder( { onUpdateOrder, instanceId, order } ) {
 	const inputId = `editor-page-attributes__order-${ instanceId }`;
 
 	return (
-		<PageAttributesCheck>
-			<label htmlFor={ inputId }>
+		<div>
```

Comment:
I would expect one or the other of a `div` wrapper, or array returns with key on each children, but not both. Alternatively, replacing the `div` with `wp.element.Fragment` and dropping the `key`. My preference would be toward one of the latter two options.

Reply:
Sorry, I was switching to the array approach, I got problems in the tests and then I used another approach but I missed the revert of key changes :( I used the Fragment approach thank you for the suggestion 👍